### PR TITLE
[W6.4h][W10-B2]Ong Kuan Yang

### PR DESCRIPTION
**Pull request by ongkuanyang**
Implements Printable interface.

### CODE REVIEW

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -24,6 +26,17 @@
     /** Offset required to convert between 1-indexing and 0-indexing.  */
     private static final int DISPLAYED_INDEX_OFFSET = 1;
 
+    /** Delimiter used for a a list of printables */
+    private static final String DELIMITER = ", ";
+
+    /** Returns a concatenated and delimited string of printables */
+    public static String getPrintableString(Printable... printables) {
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java:33]: What would be a better place to have this method? Formatter has a role to play, to format. Does this method fit in here without violating SRP?

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface for objects that supports returning a printable String
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:4]: :+1:

**Comment by okkhoy** [comment]
@ongkuanyang 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/ui/Formatter.java (hunk 1 above)
Comment (ongkuanyang) [reply src/seedu/addressbook/ui/Formatter.java:33]: The only other place I can think of is common/Utils.

**Pull request closed without merging**